packages/platforms/accton/x86-64/as5915-18x/onlp/builds/x86_64_accton_as5915_18x/module/auto/x86_64_accton_as5915_18x.yml

```diff
@@ -0,0 +1,50 @@
+###############################################################################
+#
+# x86_64_accton_as5915_18x Autogeneration Definitions.
```

Comment:
Please remove the dot/period at the end of the line.

Reply:
Removed.